app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def copy_file(command: str) -> None:
+    done_lst_path = command.split()
+
+    if done_lst_path[1] == done_lst_path[2]:
+        return
+
+    if done_lst_path[0] != "cp":
+        print("Wrong command use 'cp'.")
+        return
```

Comment:
Code Efficiency Issue: You should check if the command is 'cp' before anything else. This way, you can return early and avoid unnecessary operations if the command is not 'cp'.